### ISSUE TITLE
Add Vaccination Status option to Nearby People section

### ISSUE THREAD

**Issue by ted107mk** [microcovid/microcovid#853]
New research shows that Pfizer & AstraZeneca are effective at reducing transmission rate, in addition to the intended use case of preventing symptomatic cases. Happy to open a PR that adds "Saccination Status" to Nearby People section, potentially in "Risk Profile" section, but also perhaps in its own section. Fully vaccinated Wwould add multiplier of .6 to risk.  

"Among over 550,000 residential households in the UK between January to March 2021, the overall SARS-CoV-2 secondary attack rate was 5.7% in household where the index case had received at least one dose of the Oxford-AstraZeneca and 6.3% in households where the index case had received the Pfizer-BioNTech vaccine more than 21 days before testing positive. By contrast, the overall secondary attack rate in households with unvaccinated index cases was 10%."

https://depts.washington.edu/pandemicalliance/2021/04/29/impact-of-vaccination-on-household-transmission-of-sars-cov-2-in-england/

**Comment by beshaya** [comment]
Oh, this is really neat, thanks for bringing it to our attention! A cursory glance at that data is somewhat concerning, since prior literature found a household SAR of 30%; are they controlling for vaccination status of other household members?

**Comment by beshaya** [comment]
Skimmed the paper, they don't control for whether the household members were vaccinated. I suspect that there's a decent correlation between one member of a household being vaccinated with other members being vaccinated (similar socioeconomic status and attitude towards vaccines), which would also explain this result.

**Comment by beshaya** [comment]
I wouldn't be surprised if something like this were true, though; there seems to be a ~3x reduction in severe cases per case with pfizer and a ~6x reduction with AZ. Severe cases are more contagious.

**Comment by ted107mk** [comment]
"Households in which the index case was vaccinated 1-14 days after testing positive for
COVID-19 were also excluded, as were all contacts who had been vaccinated prior to the
index case testing positive."

Sounds like they removed any vaccinated household members from the dataset.

**Comment by beshaya** [comment]
Ok, that's quite promising. I'd like to see that paper get peer reviewed + another study corroborating the effect before we add it.

In the meanwhile, I'm actively working on getting vaccine data injested so that we can predict the risk profile of an "average vaccinated" person. Would you be interested in opening a PR that adds a vaccine selector under the "Risk Profile" question?

**Comment by beshaya** [comment]
> Duplicate of #808?

Surprisingly, no, it's looking at a different phenomena.

**Comment by ted107mk** [comment]
Yes I would be interested! I have some time off work and would love to contribute. 

Is this work tracked in https://github.com/microcovid/microcovid/issues/808#issuecomment-825960564? Let me know which of those parts (or all of them) you need help with

**Comment by beshaya** [comment]
Awesome, excited to have more help!

A good starting place would be to add a selector below the "Risk Profile"
selector:
https://github.com/microcovid/microcovid/blob/main/src/components/calculator/PersonRiskControls.tsx#L153

The selector would be for "Their vaccine: Are they fully vaccinated?" with
the options of "Yes", "No" and "I don't know." You'll want to use a
segmented control (similar to the masks selector).

**Comment by kevinburke** [comment]
I'm having trouble following this discussion. Any chance someone can summarize what you think about how vaccination changes transmission risk?

I'm vaccinated; my 10 month old son is not; I don't think 30% is the right number for household transmission risk but I don't know what the right number is. I'm happy to open a new ticket if you think that's appropriate.

**Comment by beshaya** [comment]
Sorry for the long cycle on this, here's my analysis:

Our current method for computing the riskiness of an mRNA vaccinated individual is:
    - Assume 20% of infections among unvaccinated individuals are asymptomatic and unreported. ([Byambasuren et al](https://jammi.utpjournals.press/doi/10.3138/jammi-2020-0030))
    - Assume 50% of infections among vaccinated individuals are asymptomatic and therefore by definition not part of the vaccine efficacy vs symptomatic covid. ([CDC](https://www.cdc.gov/mmwr/volumes/70/wr/mm7013e3.htm))
    - Assume asymptomatic infections are 40% as infectious as symptomatic ones ([Byambasuren et al](https://jammi.utpjournals.press/doi/10.3138/jammi-2020-0030))
    - Assume unvaccinated individuals are reported to get covid at rate Ru (this cancels out)
    - Call the efficacy of the vaccine vs symptomatic covid Ve,s
    - Infectiousness of unvaccinated individual = Ru + 0.2 * 0.4 * Ru = 1.08 Ru 
    - Infectiousness of vaccinated individual = Ru *  Ve,s + 0.4 * Ru * Ve,s = 1.4 Ve,s * Ru
    - Relative infectiousness = 1.4 Ve,s * Ru / 1.08 Ru = 1.3 Ve,s
    
If we use the 0.6 overall infectiousness modifier found by this study instead of the 0.4 for asymptomatic, we would get
    - Infectiousness of unvaccinated = 1.2 Ru
    - Infectiousness of vaccinated = 0.6 * 2 * Ve,s * Ru
    - Relative infectiousness = 1.2 * Ve,s

So this paper would only change the risk by 8%, although it would shift some of the reduction from person risk to activity risk.

**Comment by beshaya** [comment]
@kevinburke 30% is correct for transmission from you to your son. 3% from your son to you. The current spreadsheet handles this if you enter your vaccine. 

This is going to increase when we release numbers for the Delta variant. Currently looking at 38% from you to your son and 6% from your son to you.
https://khub.net/documents/135939561/405676950/Increased+Household+Transmission+of+COVID-19+Cases+-+national+case+study.pdf/7f7764fb-ecb0-da31-77b3-b1a8ef7be9aa
https://www.medrxiv.org/content/10.1101/2021.05.22.21257658v1
